### ISSUE TITLE
No control possible - Kodi 18

### ISSUE THREAD

**Issue by Natascha303** [kszaq/my-addons#8]
Hello,

Iam using your brillant addon with kodi 17.

However, i have installed adamg's LE9 and your addon does not work anymore. I can view the screen but cannot control it with the keys. Maybe you can fix this?

Many thanks

**Comment by Wuzaa** [comment]
Yes! My favorite utility for LE. Would be great to have for LE9. Would definatley donate for this if possible. Thx!

**Comment by lameventanas** [comment]
I have the same problem here.

But it looks like the process is writing the key events to /dev/uinput correctly, so this might be a problem with Kodi itself not reading the events from there.

**Comment by dmnc-net** [comment]
My problem is opposite, after LE9 I'm able to control (I can hear UI sounds) but I'm getting a small black window. I thought it is a problem with framebuffer handling but strange that other users are getting image. I'm using WeTek Play 2 (probably S905H).

Unfortunately I think this addon is dead, am I right @kszaq ?
